### PR TITLE
FIX: Account schema fingerprint

### PR DESCRIPTION
**Pull request by lumberj**
Adds unit test for validator

### CODE REVIEW

**Comment by gip** [comment]
LGTM

File: test/validatorSpec.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  beforeEach(function () {
+    this.validator = new Validator()
+    this.validator.loadSharedSchemas()
+  })
+
+  describe('create()', function () {
+    it('validates Account', function () {
+      const validate = this.validator.create('Account')
+      const account = fixtures.account
+      expect(validate(account).valid).to.be.ok
```
Comment (clark800) [inline test/validatorSpec.js]: Try changing `.to.be.ok` to `.to.be.a.pony`

**Comment by clark800** [comment]
LGTM

File: test/validatorSpec.js (hunk 1 above)
Comment (lumberj) [inline test/validatorSpec.js]: wut

**Pull request merged**